CHANGELOG.md

```diff
@@ -1,105 +1,145 @@
 - Fixing math snippet (thanks to [Zach Tidwell](https://github.com/hzach)).
-- \begin{lstlisting} syntax highlighting fix (thanks to Martin Molnar).
+- Grammar for `lstlisting` environment ([#41](https://github.com/area/language-latex/issues/41) and [#43](https://github.com/area/language-latex/pull/43)).
+
+## [0.6.1] - 2015-04-14
+### Fixed
+- Missing backslashed in snippets ([#30](https://github.com/area/language-latex/issues/30) and [#32](https://github.com/area/language-latex/pull/32)).
+
+## [0.6.0] - 2015-04-11
+__Disclaimer__: Version 0.5.0 is missing due to ower's mistake. See [`3d233d1`](https://github.com/area/language-latex/commit/3d233d1adf72d693cece41ccb55b83e983d8308c) for some details.
+
+### Added
+- Grammar for `luacode` and `luacode*` environment ([#20](https://github.com/area/language-latex/pull/20)).
+
+### Changed
+- Rename the depricated `scoped-properties` directory to `settings`.
+
+## [0.4.1] - 2014-09-03
+### Fixed
+- Malformed indentation in snippets file ([#16](https://github.com/area/language-latex/issues/16)).
+
+## [0.4.0] - 2014-09-02
+### Added
+- Snippet for generic environment.
+- Snippet for Beamer columns.
 
+### Changed
+- Utilize snippets' multi-caret feature.
 
-## 0.6.1
-* Backslash issue fix in snippets (thanks to [Thomas Johansen](https://github.com/thomasjo)).
+## [0.3.0] - 2014-03-14
+### Added
+- Enable Toggle Line Comment feature ([#7](https://github.com/area/language-latex/issues/7)).
+- Snippets for math ([#9](https://github.com/area/language-latex/pull/9)).
 
-## 0.6.0
-* Settings folder modification to fit new atom API (thanks to [Zhen-Huan (Kenny) Hu](https://github.com/akanosora)).
-* Minor bugfix.
+### Fixed
+- Grammar for `\label` and `\ref` ([#6](https://github.com/area/language-latex/issues/6)).
 
-## 0.4.1
-* Bugfix `0.4.0` ([#16](https://github.com/area/language-latex/issues/16))
+## [0.2.0] - 2014-03-11
+### Added
+- Snippets for itemizations ([#3](https://github.com/area/language-latex/pull/3)).
 
-## 0.4.0
-* Updated snippets with the use of new placeholder duplication feature ([atom/snippets#17](https://github.com/atom/snippets/issues/17)).
-* New _columns_ snippets for latex beamer (thanks to [Gwnl](https://github.com/Gwnl)).
+### Changed
+- Simplify snippet regex for section headings and math display ([#2](https://github.com/area/language-latex/pull/2) and [#3](https://github.com/area/language-latex/pull/3)).
 
-## 0.3.0
-* _Math_ snippets added (thanks to [Lucas Clemente](https://github.com/lucas-clemente)).
-* Minor bugfix.
+### Fixed
+- Unwanted line breaks in `tabular` environment ([#4](https://github.com/area/language-latex/issues/4) and [#5](https://github.com/area/language-latex/pull/5)).
 
-## 0.2.0
-* Snippets improvement.
+## 0.1.0 - 2014-02-27
+- Initial release.
 
-## 0.1.0
-* Initial release.
+[Unreleased]: https://github.com/area/language-latex/compare/v0.1.0...HEAD
```

Comment:
Is this right? I don't see much value in comparing the unreleased changes with the original release.

Reply:
This was a typo also; it should be a comparison between v1.1.1 and HEAD.